### PR TITLE
Clarify installation page

### PR DESCRIPTION
**Pull request by simonrw**
Add some clarity to the getting started page.

- Make getting started sections consistent community vs pro
- Include section on what the CLI does
- Remove the word "just"
- Add curl example commands

### CODE REVIEW

File: content/en/getting-started/api-key.md [hunk 1]
```diff
@@ -30,13 +30,15 @@ LocalStack expects your API key to be present in the environment variable `LOCAL
 {{< tabpane >}}
 {{< tab header="macOS/Linux" lang="shell" >}}
 export LOCALSTACK_API_KEY=<YOUR_API_KEY>
-localstack start -d
+localstack start
```
Comment (alexrashed) [inline content/en/getting-started/api-key.md:33]: I guess it makes sense to not start in detached mode here, but then we should also change it in line 36 (instructions for windows).

File: content/en/getting-started/installation.md [hunk 2]
```diff
@@ -58,10 +61,27 @@ If you have problems with permissions in MacOS X Sierra, install with `python3 -
...
+You can download the respective binary for your architecture directly:<br>
 {{< cli-binary-download os="linux" >}}
 </p>
 <p>
+or use this <code>curl</code> command:
+</p>
+<p>
+For <i>amd64</i>:
+{{< command >}}
+$ curl -Lo ~/Downloads/localstack-cli-{{< localstack-latest-version >}}-linux-amd64-onefile.tar.gz \
```
Comment (alexrashed) [inline content/en/getting-started/installation.md:73]: nit: I think we should not assume that there is a `Downloads` folder in the home directory of the user, but just download it into the current working directory.

File: content/en/getting-started/installation.md [hunk 3]
```diff
@@ -58,10 +61,27 @@ If you have problems with permissions in MacOS X Sierra, install with `python3 -
...
+    https://github.com/localstack/localstack-cli/releases/download/v{{< localstack-latest-version >}}/localstack-cli-{{< localstack-latest-version >}}-linux-amd64-onefile.tar.gz
+{{< / command >}}
+</p>
+<p>
+Or <i>arm64</i>:
+{{< command >}}
+$ curl -Lo ~/Downloads/localstack-cli-{{< localstack-latest-version >}}-linux-arm64-onefile.tar.gz \
```
Comment (alexrashed) [inline content/en/getting-started/installation.md:80]: nit: Same as above.

File: content/en/getting-started/installation.md [hunk 4]
```diff
@@ -58,10 +61,27 @@ If you have problems with permissions in MacOS X Sierra, install with `python3 -
...
 <p>
-You can just download the respective binary for your architecture directly:<br>
+You can download the respective binary for your architecture directly:<br>
 {{< cli-binary-download os="linux" >}}
 </p>
 <p>
+or use this <code>curl</code> command:
+</p>
+<p>
+For <i>amd64</i>:
```
Comment (alexrashed) [inline content/en/getting-started/installation.md:71]: nit: I think we should use the same architecture references as we use for the download buttons.
```suggestion
For `x86-64`:
```

File: content/en/getting-started/installation.md [hunk 5]
```diff
@@ -58,10 +61,27 @@ If you have problems with permissions in MacOS X Sierra, install with `python3 -
...
+{{< command >}}
+$ curl -Lo ~/Downloads/localstack-cli-{{< localstack-latest-version >}}-linux-amd64-onefile.tar.gz \
+    https://github.com/localstack/localstack-cli/releases/download/v{{< localstack-latest-version >}}/localstack-cli-{{< localstack-latest-version >}}-linux-amd64-onefile.tar.gz
+{{< / command >}}
+</p>
+<p>
+Or <i>arm64</i>:
```
Comment (alexrashed) [inline content/en/getting-started/installation.md:78]: nit: I think we should use the same architecture references as we use for the download buttons.
```suggestion
Or `ARM64`:
```

**Review by alexrashed — APPROVED** [review]
Awesome! Thanks for proactively updating the docs! I only found a few nitpicks. 💯

File: content/en/getting-started/installation.md (hunk 2 above)
Comment (simonrw) [reply content/en/getting-started/installation.md:73]: I agree, I was making the curl command consistent with the tar unpack instructions. I'll update both to use the CWD :+1:

File: content/en/getting-started/api-key.md (hunk 1 above)
Comment (simonrw) [reply content/en/getting-started/api-key.md:33]: This was made to be consistent with the pro instructions. I'll update the windows docs as well.

**Pull request merged**